doc-content/enterprise-only/processes/interacting-with-processes-viewing-task-variables-basic-filters-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='interacting-with-processes-viewing-task-variables-basic-filters-proc']
+= Viewing task variables using basic filters
+
+{CENTRAL} provides basic filters to view task variables in *Manage Tasks* and *Task Inbox*. You can view the task variables of the task as columns using *Show/hide columns*.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Tasks* or  go to *Menu* -> *Track* -> *Task Inbox*.
+. On the *Task Inbox* page, click the filter icon on the left of the page to expand the *Filters* panel
+. In the *Filters* panel, select the *Task Name*.
++
+The filter is applied on the current task list.
```

Comment:
Question - is the filter acting on the current UI view or is it being applied to the task list?

s/applied on/applied to